### PR TITLE
Remove `--user` as per user requested.

### PR DESCRIPTION
**Pull request by xuzhao9**
Addresses user comment on https://github.com/pytorch/benchmark/pull/2123

### CODE REVIEW

**Comment by cota** [comment]
LGTM. This should fix my issue -- I can double-check on my side while we wait for an Approval.

**Comment by cota** [comment]
> I can double-check on my side while we wait for an Approval.

Confirmed. This fixes the issue for me.

**Review by aaronenyeshi — APPROVED** [review]

**Pull request closed without merging**
